postatus/templates/l10n_summary.html

```diff
@@ -45,10 +44,8 @@
                     {{ item.untranslated }}
                   </td>
                   <td>
-                    <span class="perc perc-{% if bars == 20 %}good{% elif bars > 15 %}ok{% else %}bad{% endif %}">
-                      {% for chunk in range(bars) %}
-                        <span class="perc-notch">=</span>
-                      {% endfor %}
+                    <span class="perc perc-{% if item.percent == 100 %}good{% elif item.percent > 75 %}ok{% else %}bad{% endif %}">
+                      <span style="width: {{ item.percent }}%"></span>
```

Comment:
This only covers the highlighted locales. You also need to update the locales section below.